draft-rha-jose-hpke-encrypt.md

```diff
@@ -127,69 +121,86 @@ carries the output to JOSE ({{RFC7516}}, {{RFC7518}}).
 
 When the alg value is set to any of algorithms registered by this specification then the 'epk' header parameter MUST be present, and it MUST be a JSON Web Key as defined in {{EK}} of this document.
+The "ek" member of an 'epk' will contain the base64url encoded "enc" value produced by the encapsulate operation of the HPKE KEM.
+
+In all serializations, "ct" will be base64url encoded.
 
-In both modes, the header parameter 'epk' will contain the 'ek' member. The "ek" member will contain the base64url encoded "enc" value produced by the encapsulate operation of the HPKE KEM.
+The two modes (Integrated Encryption, and Key Encryption) can be distinguished by determining whether an "enc" JWE Protected Header parameter is present. 
 
-In both JWE Compact Serialization and the JWE JSON Serialization, "ct" and "enc" will be base64url encoded (see Section 7.1 and 7.2 of {{RFC7518}}), since JSON lacks a way to directly represent arbitrary octet sequences.   
+If "enc" is present in the JWE Protected Header, the mode is Key Encryption; otherwise, the mode is Integrated Encryption.
 
-The two modes (Integrated Encryption, and Key Encryption) can be distinguished by determining whether an "enc" member is present in the protected header. 
+Interested readers will observe this is due to all recipients using the same JWE Protected Header when JSON Serializations are used, as described in {{Section 7.2.1 of RFC7516}}.
 
-If the "enc" member exists, it is a Key Encryption mode; otherwise, it is a Integrated Encryption mode.
+We provide the following table for additional clarity:
 
-## HPKE Encryption with SealBase
+| Name                   | Recipients | Serializations | Content Encryption Key | Similar to
+|---
+| Integrated Encryption  | 1          | Compact        | Derived from HPKE      | Direct Key Agreement
+| Key Encryption         | Many       | JSON           | Encrypted by HPKE      | Key Agreement with Key Wrapping
+{: #serialization-mode-table align="left" title="JOSE HPKE Serializations and Modes"}
+
+## HPKE Encryption
 
 The message encryption process is as follows. 
 
 1. The sending HPKE context is created by invoking invoking SetupBaseS() (Section 5.1.1 of {{RFC9180}}) with the recipient's public key "pkR" and "info". The HPKE specification defines the "info" parameter as a context information structure that is used to ensure that the derived keying material is bound to the context of the transaction. The SetupBaseS function will be called with the default value of an empty string for the 'info' parameter. This yields the context "sctxt" and an encapsulation key "enc". 
 
-2. If "zip" parameter is present, compression is applied to the plaintext "pt" using the specified compression algorithm. Encrypt plaintext "pt" by invoking the Seal() method (Section 5.2 of {{RFC9180}}) on "sctxt" with "aad", yielding ciphertext "ct".
```

Comment:
I removed zip

Reply:
Makes sense. I have no idea how this zip compression sneaked into the document